From ab01eed37be812a5f2848445de8e3323e3a9d367 Mon Sep 17 00:00:00 2001
From: Bernhard Frauendienst <exherbo@nospam.obeliks.de>
Date: Thu, 30 Apr 2015 22:02:01 +0200
Subject: [PATCH] Don't use deprecated automake init/options

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index fedac73..bd114f3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -3,7 +3,7 @@ dnl Created by Anjuta application wizard.
 
 AC_INIT(boblight, 2.0)
 
-AM_INIT_AUTOMAKE(AC_PACKAGE_NAME, AC_PACKAGE_VERSION)
+AM_INIT_AUTOMAKE([subdir-objects])
 AC_CONFIG_HEADERS([config.h])
 AM_MAINTAINER_MODE
 
-- 
2.3.3

